docs/content/docs/operations/deployment.md

```diff
@@ -24,18 +24,141 @@ under the License.
 
 ## Local Run with Test Data
```

Comment:
Before introducing each deployment mode, I think we need a opening paragraph to briefly introduce that there are three ways of executing the agents, and what are the differences and how to choose between them.